pennylane/transforms/__init__.py

```diff
@@ -152,3 +152,4 @@
     create_decomp_expand_fn,
     set_decomposition,
 )
+from .transpile import transpile
```

Comment:
In order for this function to appear in the [documentation build for this PR](https://pennylane--2118.org.readthedocs.build/en/2118/code/qml_transforms.html), you will also need to include it in the relevant part of the docstring above!

Note that you can get to the documentation build by clicking on the 'details' link to the right of the 'ReadTheDocs' CI check:

![image](https://user-images.githubusercontent.com/2959003/150685251-ccb70dda-fd2d-41ab-9eda-c62613c0a6af.png)

Reply:
I will include this under "Transforms for circuit compilation"; then running make docs also works and the function appears in the documentation.